README.md

```diff
@@ -16,6 +18,22 @@ $ga = new Google_Chrome_Labs\WP_Third_Parties\Third_Parties\Google_Analytics(
 add_action( 'wp_loaded', array( $ga, 'add_hooks' ) );
 ```
 
+See the [Google Analytics JSON schema](https://github.com/GoogleChromeLabs/third-party-capital/blob/main/data/google-analytics.json) for the full list of supported arguments.
+
+### Google Tag Manager
+
+```php
+$gtm = new Google_Chrome_Labs\WP_Third_Parties\Third_Parties\Google_Tag_Manager(
+	array(
+		'id' => 'GTM-...', // Replace this with your actual Google Tag Manager ID.
+	)
+);
+
+add_action( 'wp_loaded', array( $gtm, 'add_hooks' ) );
```

Comment:
reviewing the entire doc I noticed they all include a hook line like `add_action( 'wp_loaded', array( $yte, 'add_hooks' ) );` except "Google Maps Embed". Should it also include the hook line? If not maybe a single line explaining why these are included in the other examples but not this one.

Reply:
@adamsilverstein Great point! I clarified this in https://github.com/GoogleChromeLabs/wp-third-parties/pull/3/commits/7c43462c962cfe6a5652ad523f29008dd2a5a005.

The reason Google Maps Embed doesn't need to add hooks is that there are no assets (scripts or stylesheets) needed for it. You could technically still include the same hook callback, but it just wouldn't do anything. So if you know that already, you can omit it, that's why the snippet here does that.